src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4301,7 +4345,7 @@ private void VisitMemberAccess(BoundExpression node, BoundExpression receiverOpt
                     if (!resultType.IsValueType || resultType.IsNullableType())
                     {
                         int containingSlot = getReceiverSlot();
-                        int slot = (containingSlot < 0) ? -1 : GetOrCreateSlot(member, containingSlot);
+                        int slot = (containingSlot < 0) ? -1 : GetOrCreateSlot(member, containingSlot, createIfMissing: false);
```

Comment:
>createIfMissing: false [](start = 103, length = 22)

I am not sure if passing this value is going to have any useful effect. If we are not interested in allocating new slots, perhaps we should use VariableSlot API instead. But do we really want to skip allocating the slot and fail to adjust ```resultType```?